clients/js/near.ts

```diff
@@ -0,0 +1,124 @@
+import { impossible, Payload } from "./vaa";
+import { NETWORKS } from "./networks";
+import { CONTRACTS } from "@certusone/wormhole-sdk";
+
+const BN = require("bn.js");
+const nearAPI = require("near-api-js");
+
+export async function execute_near(
+  payload: Payload,
+  vaa: string,
+  network: "MAINNET" | "TESTNET" | "DEVNET"
+) {
+  let n = NETWORKS[network]["near"];
+  let contracts = CONTRACTS[network]["near"];
+
+  let account: string;
+
+  switch (payload.module) {
+    case "Core":
+      if (contracts.core === undefined) {
+        throw new Error("Core bridge not supported yet for near");
+      }      
+      account = "wormhole." + n.baseAccount
```

Comment:
so on mainnet this is `wormhole.wormhole.mainnet`?

Reply:
So, the 3 accounts would be:
wormhole.wormhole.mainnet
token.wormhole.mainnet
nft.wormhole.mainnet